controllers/Transactions/transationsUpdateController.js

```diff
@@ -0,0 +1,33 @@
+const createHttpError = require("http-errors");
+const { catchAsync } = require("../../helpers/catchAsync");
+const { endpointResponse } = require("../../helpers/success");
+const { ErrorObject } = require("../../helpers/error");
+const { Transaction } = require("../../database/models");
+
+module.exports = {
+  updateTransactionById: catchAsync(async (req, res, next) => {
+    try {
+      const { id } = req.params;
+
+      const transactionById = await Transaction.findOne({ where: { id } });
+
+      if (!transactionById) {
+        throw next(new ErrorObject(" transaction by Id not found ", 404));
```

Comment:
throw the error outside the next function , next function is used in catch

Reply:
ya se resolvieron los cambios